### DISCUSSION TITLE
Issue in reading locally stored Iceberg table

### DISCUSSION THREAD

**Discussion by rajasekarv** [apache/doris#13768] [A - General / Q&A]
I am testing Doris in single node deployment. 

I have created a sample ICEBERG table stored in local path and the hive metastore is also running locally. I am able to create an external table in Doris and able to see the schema using "DESCRIBE" command. However, while reading the data, it throw**s "ERROR 1105 (HY000): errCode = 2, detailMessage = Not supported storage type: file "**

Steps to reproduce:

1. In spark-sql shell - "create table sample using iceberg  tblproperties("write.parquet.compression-codec" = "snappy") location 'file:///root/dev/temp/sample'  select 1 as id"
2. In mysql client - create external table sample ENGINE=ICEBERG PROPERTIES ( "iceberg.database" = "default", "iceberg.table" = "sample", "iceberg.hive.metastore.uris" = "thrift://localhost:9083", "iceberg.catalog.type" = "HIVE_CATALOG" )

MySQL [icetest]> describe sample;
+-------+------+------+------+---------+-------+
| Field | Type | Null | Key  | Default | Extra |
+-------+------+------+------+---------+-------+
| id    | INT  | Yes  | true | NULL    |       |
+-------+------+------+------+---------+-------+

MySQL [icetest]> select * from sample;
ERROR 1105 (HY000): errCode = 2, detailMessage = Not supported storage type: file

SPARK Version: 3.3.0
ICEBERG Version: org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.0.0
Doris Version: 1.1.3(1 fe, 1 be both running on same machine)

**Comment by morningman** [comment]
Doris does not support file in "local", most of users store file on hdfs or s3.
But we may support it in v1.2 for easy testing in local env.

**Reply by rajasekarv** [reply]
Thought so. Thank you

**Reply by rajasekarv** [reply]
I tried reading iceberg table from azure block storage. Now it shows filesystem is not supported. I have added both Hadoop-azure and hadoop-azure-datalake jars to the fe/lib folder and restarted the fe

**Reply by rajasekarv** [reply]
@morningman
